### PR TITLE
LPS-198757

### PR DESCRIPTION
**Pull request by TamyrisTorres**
LPS-198757

### CODE REVIEW

**Comment by TamyrisTorres** [comment]
ci:test:sf

**Comment by TamyrisTorres** [comment]
ci:test:relevant

File: modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/BaseUpgradeCheck.java [hunk 1]
```diff
@@ -24,15 +24,16 @@
...
+		String fileName, String message, List<String> parameterList,
+		String[] parameterTypes) {
 
 		if (parameterList.size() != expectedParametersSize) {
 			return false;
 		}
 
 		if (!hasParameterTypes(
-				javaMethodContent, javaMethodContent,
+				content, fileContent, fileName,
```
Comment (natocesarrego) [inline modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/BaseUpgradeCheck.java]: Good job in the commit renaming, @TamyrisTorres! :wink:

File: modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/JavaConstructorParametersCheck.java [hunk 2]
```diff
@@ -303,8 +303,8 @@ private String _fixPassedInVariables(
 	}
 
 	private String _fixPassedInVariables(
-		String absolutePath, String content, List<JavaParameter> parameters,
-		String fileContent) {
+		String absolutePath, String content, String fileName,
+		List<JavaParameter> parameters, String fileContent) {
```
Comment (natocesarrego) [inline modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/JavaConstructorParametersCheck.java]: @TamyrisTorres, is it very costly to sort all parameters alphabetically?

If not, please do it to mitigate the risk of Brian complain about it.

File: modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/JavaMapBuilderGenericsCheck.java [hunk 3]
```diff
@@ -34,15 +34,17 @@ protected String doProcess(
 			String fileContent)
 		throws IOException {
 
-		return _formatGenerics(javaTerm, fileContent);
+		return _formatGenerics(fileName, javaTerm, fileContent);
```
Comment (natocesarrego) [inline modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/JavaMapBuilderGenericsCheck.java]: @TamyrisTorres, is it very costly to sort all parameters alphabetically?

If not, please do it to mitigate the risk of Brian complain about it.

File: modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaFDSDataProviderCheck.java [hunk 4]
```diff
@@ -60,33 +60,44 @@ protected String format(
 	}
 
 	private boolean _checkMethodCall(
-		String content, String javaMethodContent, String methodCall) {
+		String content, String javaMethodContent, String fileName,
+		String methodCall) {
```
Comment (natocesarrego) [inline modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaFDSDataProviderCheck.java:64]: @TamyrisTorres, please sort parameters alphabetically

**Review by natocesarrego — CHANGES_REQUESTED** [review]

**Comment by TamyrisTorres** [comment]
@natocesarrego resolved

File: modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/JavaUpgradeModelPermissionsCheck.java [hunk 5]
```diff
@@ -199,11 +204,18 @@ private String _getModelPermissionsImplementation(
 	}
 
 	private boolean _isServiceContextMethodCall(
-		String methodCall, String fileContent, String variableName) {
+			String methodCall, String fileContent, String fileName,
+			String variableName)
```
Comment (natocesarrego) [inline modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/JavaUpgradeModelPermissionsCheck.java:208]: @TamyrisTorres, can you please sort alphabetically?

**Review by natocesarrego — CHANGES_REQUESTED** [review]

File: modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/JavaUpgradeModelPermissionsCheck.java (hunk 5 above)
Comment (TamyrisTorres) [reply modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/JavaUpgradeModelPermissionsCheck.java]: @natocesarrego resolved

**Comment by TamyrisTorres** [comment]
ci:test:relevant

**Comment by TamyrisTorres** [comment]
ci:test:sf

**Comment by TamyrisTorres** [comment]
ci:test:relevant

**Comment by TamyrisTorres** [comment]
ci:test:sf

**Comment by TamyrisTorres** [comment]
ci:test:relevant

**Review by natocesarrego — APPROVED** [review]

**Comment by natocesarrego** [comment]
ci:test:hotel

**Comment by natocesarrego** [comment]
ci:test:relevant

**Comment by natocesarrego** [comment]
@TamyrisTorres, your pull is approved!

I'll re-run the test relevant suite since its last run failed to build the portal. Once it completes then I'll ask a QA to analyse it to make sure we're not introducing regressions. :wink: 

Thank you!

**Comment by natocesarrego** [comment]
ci:forward

**Comment by pyoo47** [comment]
@natocesarrego You `ci:forward` attempt failed because the branch failed to rebase.

```
[beanshell] com.liferay.jenkins.results.parser.GitWorkingDirectory$GitWorkingDirectoryRuntimeException: Runtime exception  occurred in liferay-portal
[beanshell] Unable to rebase ci-forward-LPS-198757-pr-1398-sender-TamyrisTorres to master
[beanshell] Rebasing (1/5)
[beanshell] Rebasing (2/5)
[beanshell] 
[beanshell] Rebasing (3/5)
[beanshell] 
[beanshell] Rebasing (4/5)
[beanshell] 
[beanshell] Rebasing (5/5)
[beanshell] 
[beanshell] error: could not apply eaa3e3b6194ee... LPS-198757 Fix in upgrade files
```

**Comment by natocesarrego** [comment]
@TamyrisTorres, can you please rebase to fix it?

Then, I'll manually forward to Brian

**Comment by natocesarrego** [comment]
Thanks for your proactive analysis, @pyoo47 :slightly_smiling_face:

**Comment by TamyrisTorres** [comment]
@natocesarrego resolved

**Comment by natocesarrego** [comment]
Manually forwarded to Brian: https://github.com/brianchandotcom/liferay-portal/pull/143706

**Pull request closed without merging**